src/engine/Common.h

```diff
@@ -0,0 +1,9 @@
+#ifndef GOLEM_COMMON_H
+#define GOLEM_COMMON_H
+
+#include "graph/ChcGraph.h"
+#include "Witnesses.h"
+
+VerificationResult solveTrivial(ChcDirectedGraph const & graph);
```

Comment:
Hmm, should it be a separate "engine" though? Maybe `utils`? Wdyt?

Reply:
Yes, this is basically `utils` for all engines. What do you suggest? Rename the file?

Reply:
Maybe create an "utils" folder in "engines" and reuse it? Wdyt?

Reply:
I am not very keen on expanding the folder structure at the moment. Maybe if there are more things to put there in the future...